packages/Chem/src/OCL-service/OCL-service.ts

```diff
@@ -13,13 +13,16 @@ export class OCLService {
     molCol:DG.Column, argList: any[], op: OCLServiceCall,
   ): Promise<{[key: string]: Array<number>}> {
     const molList = molCol.toList();
+    // detect if the column is smiles or molblock notation
+    const notationType = DG.chem.isMolBlock(molList[0]) ? 'molblock' : 'smiles';
```

Comment:
I guess we have somewhere constants for these types?
if not, maybe it's better to create it